String/BuddyStrings

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
It's strange that this file doesn't have a `.swift` file extension. However, you included `.swift` in the README. I believe that the link is broken.